selftests/unit/test_nrunner.py

```diff
@@ -250,54 +250,6 @@ def test_runner_exec_test_fail(self):
         self.assertEqual(last_result['returncode'], 1)
         self.assertIn('time', last_result)
 
-    def test_runner_python_unittest_ok(self):
```

Comment:
Please squash this change to wherever the behavior changes, for bisectability reasons.